test/parallel/test-trace-events-module.js

```diff
@@ -0,0 +1,65 @@
+'use strict';
+const common = require('../common');
+const assert = require('assert');
+const cp = require('child_process');
+const path = require('path');
+const fs = require('fs');
+const tmpdir = require('../common/tmpdir');
+
```

Comment:
Can you please add a small comment explaining what this test does? are we validating trace_events  with perf_hooks? I would prefer to have an independent test.

Reply:
It's using the fact that perf_hooks emits a very small number of very predictable trace events. The v8 and node.async_hooks trace events tend to be more unpredictable.